startvscode.cmd

```diff
@@ -0,0 +1,32 @@
+@ECHO OFF
+SETLOCAL
```

Comment:
Let's update the `BuildFromSource.md` file to indicate that users will need to run this script before launching their projects in VS Code.

Reply:
I have updated `BuildFromSource.md`, but please let me know if there are more details to put on!